numpy/_core/tests/test_nditer.py

```diff
@@ -3322,9 +3322,10 @@ def test_debug_print(capfd):
     |   BufferSize: 50
     |   Size: 5
     |   BufIterEnd: 5
+    |   BUFFER CoreSize: 5
     |   REDUCE Pos: 0
-    |   REDUCE OuterSize: 10
-    |   REDUCE OuterDim: 1
+    |   BUFFER Reduce outersize: 10
+    |   BUFFER OuterDim: 1
```

Comment:
I suppose the reduce outersize is actually also always valid now.  Which would be useful for advancing the iterator.
Should rename/reorganize, but would add churn elsewhere too probably.